infrastructure/aws/s3.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_s3_bucket" "user_data_bucket" {
+  bucket = "${var.project_name}-${terraform.workspace}-user-data"
+}
```

Comment:
Lets use the our s3 module here I think?

https://github.com/i-dot-ai/i-ai-core-infrastructure/blob/main/instances/platform/s3.tf